src/containerapp/azext_containerapp/_up_utils.py

```diff
@@ -403,19 +487,43 @@ def run_acr_build(self, dockerfile, source, quiet=False, build_from_source=False
         image_name = self.image if self.image is not None else self.name
         from datetime import datetime
 
-        now = datetime.now()
-        # Add version tag for acr image
-        image_name += ":{}".format(
-            str(now).replace(" ", "").replace("-", "").replace(".", "").replace(":", "")
-        )
-
         self.image = self.registry_server + "/" + image_name
 
+        now = datetime.now()
+        tag_now_suffix = str(now).replace(" ", "").replace("-", "").replace(".", "").replace(":", "")
+
         if build_from_source:
-            # TODO should we prompt for confirmation here?
             logger.warning("No dockerfile detected. Attempting to build a container directly from the provided source...")
-            self.build_container_from_source(image_name, source)
+
+            try:
+                # First try to build source using buildpacks
+                # Temporary fix: using run time tag as customer image tag
+                # Waiting for buildpacks side to fix this issue: https://github.com/buildpacks/pack/issues/1753
```

Comment:
Ah I do apologize for linking wrong issue page. I've updated it just now with the linting changes in the fork. 

Yes, the issue from buildpack was the rebasing doesn't correctly update the info provided when running `pack inspect`. Usually, we would see the run image change after running `pack rebase --run-image`. Therefore, instead of actively looking for the image inspection results, the patcher will updates and also looks for the run image specified in user image tag. 

Because of the issue we experienced on Friday where we build an image using newest run image (7.0.5), the image inspection is still showing run image version 7.0 even though the buildpack log states pulling and using 7.0.5. Therefore, we add this as a temporary fix to use the same run image tag we use from Oryx MCR so that the patcher can recognize the correct run image.

Sorry for the long comment. Hope that makes some sense.